doc/source/faq.rst

```diff
@@ -12,6 +12,30 @@ an issue on GitHub or talk to us on the Slack team or mailing list. See the
     :depth: 1
     :local:
 
+
+How can I speed up image production for composites that need resampling?
```

Comment:
```suggestion
How can I speed up creation of composites that need resampling?
```

I'd like to avoid language that assumes we're generate "images". I don't want to confuse people who aren't generating images (PNG, JPEG) as their final output that this is only an option when writing to disk. Some people may also not consider creating interactive plots as generating images. Also, this doesn't only apply to RGB/A composites but any composite and any modifier.